web-app/src/hoc/Layout/container.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import {
+  faUser,
+  faTachometerAlt,
+  faUnlockAlt,
+} from '@fortawesome/fontawesome-free-solid';
+
+export default Wrapped =>
+  class extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        menuItems: [
+          {
```

Comment:
This would be better placed in a `navConfig.js` file or something. Use state for stuff that you anticipate to change while the user is using the app. I don't think this will be changing.

Because of this, the whole container isn't needed.

Reply:
Yep so just ignore this feedback - I can see why you've done this to support changes in the future, nice work man